scripts/eval/eval.py

```diff
@@ -301,6 +303,10 @@ def main(cfg: DictConfig) -> Tuple[List[Trainer], pd.DataFrame]:
                                              'loggers',
                                              must_exist=False,
                                              default_value={})
+    fewshot_random_seed: int = pop_config(cfg,
```

Comment:
does this need to be a new top level arg? I feel like it should just the seed top level arg that already exists, and then individual icl tasks hsould be able to override it if needed. wdyt

Reply:
I don't see random_seed anywhere else? Can you point me there? 
Per task would be better imo, some tasks might vary with different random seed settings more than others and that would be more controllable.

Reply:
https://github.com/mosaicml/llm-foundry/blob/2e0a845863dc18ad8ddac71fc7f18a9940989891/scripts/eval/eval.py#L289

I'd suggest that you default to this seed, but allow overriding in the individual task configs

Reply:
ffs I searched random and random_seed but not seed lol
what you have is much better than this big pass through

Reply:
lol np, there should be docs and a dataclass :P